src/figvalidationservice/src/test/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/applicant/ApplicantServiceValidateApplicantService.java

```diff
@@ -0,0 +1,101 @@
+package ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.applicant;
+
+import ca.bc.gov.open.ords.figcr.client.api.ApplicantApi;
+import ca.bc.gov.open.ords.figcr.client.api.handler.ApiException;
+import ca.bc.gov.open.ords.figcr.client.api.model.ValidateApplicantServiceOrdsResponse;
+import ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.exception.FigaroValidationServiceException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+/**
+ * @author shaunmillargov
+ * <p>
+ * ValidateApplicantPartyId Controller test class.
+ * <p>
+ * Mocks underlying ORDS service class.
+ */
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class ApplicantServiceValidateApplicantService {
+
+
+    private static final String VALIDATION_RESULT = "result";
+    private static final String STATUS_MESSAGE = "success";
+    private static final String STATUS_CODE = "0";
+    private static final String ERROR_VALIDATION_RESULT = "error_result";
+    private static final String ERROR_STATUS_MESSAGE = "fail";
+    private static final String ERROR_STATUS_CODE = "-2";
+
+    @Mock
+    private ApplicantApi applicantApiMock;
+
+    private ApplicantServiceImpl sut;
+
+    @BeforeAll
+    public void SetUp() throws ApiException {
+
+        MockitoAnnotations.initMocks(this);
+
+        ValidateApplicantServiceOrdsResponse successResponse = new ValidateApplicantServiceOrdsResponse();
+        successResponse.setValidationResult(VALIDATION_RESULT);
+        successResponse.setStatusMessage(STATUS_MESSAGE);
+        successResponse.setStatusCode(STATUS_CODE);
+
+        ValidateApplicantServiceOrdsResponse errorResponse = new ValidateApplicantServiceOrdsResponse();
+        errorResponse.setValidationResult(ERROR_VALIDATION_RESULT);
+        errorResponse.setStatusMessage(ERROR_STATUS_MESSAGE);
+        errorResponse.setStatusCode(ERROR_STATUS_CODE);
+
+        Mockito.when(applicantApiMock.validateOrgApplicantService(Mockito.eq("1"), Mockito.anyString())).thenReturn(successResponse);
+        Mockito.when(applicantApiMock.validateOrgApplicantService(Mockito.eq("2"), Mockito.anyString())).thenReturn(errorResponse);
+        Mockito.when(applicantApiMock.validateOrgApplicantService(Mockito.eq("3"), Mockito.anyString())).thenThrow(ApiException.class);
+
+        sut = new ApplicantServiceImpl(applicantApiMock);
+
+    }
+
+    /**
+     * success
+     */
+    @Test
+    public void withValidResponseShouldReturnValidResponse() throws FigaroValidationServiceException {
+
+        ValidateApplicantServiceOrdsResponse response = sut.validateApplicantService("1", "1");
```

Comment:
again here for consistency (extracting string values to constants)?

Reply:
sounds good